forge-gui/res/cardsfolder/upcoming/gardens_of_tranquil_repose.txt

```diff
@@ -0,0 +1,11 @@
+Name:Gardens of Tranquil Repose
+ManaCost:no cost
+Types:Plane Necros
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature | Execute$ TrigChange | TriggerZones$ Command | TriggerDescription$ Suspended Animation — Whenever a creature dies, exile it. Its controller scries 1.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | Defined$ TriggeredNewCardLKICopy | RememberChanged$ True | SubAbility$ DBScry
```

Comment:
why remember

Reply:
I've tried with` SVar:X:ExiledWith$Amount` but that didn't seem to work

Reply:
yea you're right
for now Remembered is also ok if you fix cleanup and the +1